apps/server/src/utils/parser.ts

```diff
@@ -494,15 +422,16 @@ export const fileHandler = async (file: string, options: ExcelImportOptions): Pr
     if (res.data.rundown.length < 1) {
       throw new Error(`Could not find data to import in the worksheet ${options.worksheet}`);
     }
-    res.data.project = parseProject(dataFromExcel);
     res.data.userFields = parseUserFields(dataFromExcel);
 
-    await deleteFile(file);
+    deleteFile(file);
 
     return res;
   }
 
   if (file.endsWith('.json')) {
+    console.log('JSON!');
```

Comment:
console log :)

Reply:
good catch, this is fixed on the stacked PR